content/communication/index.md

```diff
@@ -51,7 +51,7 @@ The following places are not sources of truth. Treat documents and conversations
 
 #### Scheduling meetings with Google Calendar
 
-Before you start, please make sure you've [configured your Google Calendar settings](../people-ops/onboarding/general_onboarding.md#configure-google-calendar) as described in the onboarding guidelines.
+Before you start, please make sure you've [configured your Google Calendar settings](../handbook/editing/add-yourself-to-team-page.md) as described in the onboarding guidelines.
```

Comment:
@InesRoitman Does this link need to be updated?

Reply:
@daxmc99 Done! Thank you :)